packages/block-editor/src/components/block-switcher/test/index.js

```diff
@@ -15,13 +15,22 @@ import { copy } from '@wordpress/icons';
  * Internal dependencies
  */
 import { BlockSwitcher, BlockSwitcherDropdownMenu } from '../';
-import { act } from 'react-test-renderer';
```

Comment:
Oh, we didn't remove all of them yet? 🙂

Reply:
I think some of those fixes got lost in the thousand rebases 😅 We'll clean them up, don't worry.